[Doc] ecs add mapping table

kaisecheng opened the pull request:
A follow-up of #24
Add ECS compatibility table

karenzone reviewed (changes requested):
Hi @kaisecheng. Thank you for adding the table. We need a few changes to be consistent with the new convention we're following from [input-beats](https://www.elastic.co/guide/en/logstash/master/plugins-inputs-beats.html). ([Link to source doc](https://www.elastic.co/guide/en/logstash/master/plugins-inputs-beats.html).)  

Suggested changes:
* Add an ECS section and brief description after "Description."
* Move the new table to that section with the table title removed and a lead in sentence like: "Here’s how ECS compatibility mode affects output."
* The column headers should be "ECS disabled" and "ECS v1" 

These changes should bring this plugin up to the new standard. Thanks!

kaisecheng commented:
Good suggestion. I was thinking this ecs change is so small, and maybe we have too much description.

I moved the table and added the ECS description. Please have a look @karenzone and thanks 🌹

karenzone reviewed (approved):
LGTM. Thanks!
